sos/plugins/__init__.py

```diff
@@ -198,6 +201,15 @@ def do_cmd_output_sub(self, cmd, regexp, subst):
         if not self.executed_commands:
             return 0
 
+        if dotall and ignore_case:
+            flags = re.IGNORECASE | re.DOTALL
+        elif dotall and not ignore_case:
+            flags = re.DOTALL
+        elif ignore_case and not dotall:
+            flags = re.IGNORECASE
+        else:
+            flags = None
```

Comment:
This code will probably disappear when rewritten to use `|` but this should be written by first initialising `flags = None` and then removing the final `else`.